www/js/diary/infinite_scroll_list.js

```diff
@@ -200,6 +210,7 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
     });
     $scope.allTrips = false;
     $scope.recomputeDisplayTrips();
+    $ionicScrollDelegate.scrollBottom();
```

Comment:
Ah, so we scroll to the bottom every time we switch tabs. Maybe a comment explaining why?